src/GeneralRules/ReConsistencyCheckRule.class.st

```diff
@@ -32,5 +32,5 @@ ReConsistencyCheckRule >> initialize [
 
 { #category : #accessing }
 ReConsistencyCheckRule >> name [
-	^ 'Uses "size = 0" instead of "isEmpty"'
+	^ 'Checks for empty collection using #size instead of #isEmpty" or #isNotEmpty'
```

Comment:
I have no idea why this is here :(

Reply:
the PR was against Pharo10 wich does not yet have that change, I rebased it